### DISCUSSION TITLE
Simple JS project

### DISCUSSION THREAD

**Discussion by night2876** [Kruptein/planarally-dice#9] [Q&A]
Hi,

i'm not an expert web developer but your project is very intresting but to difficult to me. Could be possible to use this dice roller in a simple web page with simple javascript? Could you create an example in this way ...

Best Regards

M.

**Comment by Kruptein** [comment]
Hey, thanks for checking this out.

It's still in a relatively experimental phase, which is the reason why you might have some difficulties with getting it to work on your own. My apologies for that!

Have you tried checking out any of the examples in the examples folder? (I will admit that they might no longer work due to recent changes, I'll try to revisit them in the short future)

**Reply by night2876** [reply]
Hi,

yeah i see all the example but is all in typescript and i know only html and javascript. I understood that the file .json are the dependancy to "compile" the project but as i said i'm a flat web developer and i know only html and plain javascript your coding is more advanced ... and i don't understood it very well or how i can port it to my simple world.

Thanks for the answer

M.
